com/edocti/jintro/lab02/FunctionSignature.java

```diff
@@ -0,0 +1,148 @@
+package com.edocti.jintro.lab02;
+
+import java.util.Scanner;
+
+public class FunctionSignature {
+
+
+    //1. O functie care calculeaza suma elementelor unui array de numere intregi
+    public static int sum(int[] a) {
+    	if (a == null || a.length == 0) {
+            return 0;
+        }
```

Comment:
It's one space too far to the right. This is not very important (apparently), but I insist on them in order for Ctrl-Shift-F to become common practice.